app/src/main/res/values/strings.xml

```diff
@@ -778,16 +778,16 @@
     <string name="activity_result">Activity result</string>
     <string name="result_code">Result code</string>
     <string name="action">Action</string>
-    <string name="mime_type">MIME Type</string>
+    <string name="mime_type">Media type</string>
```

Comment:
https://en.wikipedia.org/wiki/Media_type this is the correct one.
MIME anything just confuses it with the one used for e-mail.

Reply:
As I've said earlier, I use Android terminologies here for familiarity. There's not really anything to argue here.

Reply:
This predates the change
https://developer.android.com/reference/android/webkit/MimeTypeMap
Media type used here
https://developer.android.com/guide/topics/media/camera

Android sources are all over the place.

It is an IANA term.

Reply:
It does describe internal nomenclature and not the function. See https://muntashirakon.github.io/AppManager/guide/interceptor-page.html#mime-type and https://developer.android.com/guide/components/intents-filters

Android's use of MIME type is not the same as RFC definition.

Reply:
So since Android Media Type is what MIME type (now Media type) was back in 1996, it is now only subclass?

I don't understand why users need to be subject to a word that makes no sense, and at a technical level makes no sense(?). But I'll revert it, no problem.

Reply:
It makes sense because it's used in the documentation of [Intent](https://developer.android.com/reference/android/content/Intent). If you have difficulties in digesting it, I suggest you open an issue in the Google issue tracker so that they can fix the docs.